ipxe/ipxe.go

```diff
@@ -0,0 +1,20 @@
+package ipxe
+
+import (
+	"bytes"
+	"html/template"
+)
+
+func GenerateTemplate[data any](d data, script string) (string, error) {
```

Comment:
This shouldn't need to be generic because `t.Execute() accepts the `any` type.


```suggestion
func GenerateTemplate(d any, script string) (string, error) {
```

Reply:
ah thanks. updated